integration-requirements.txt

```
@@ -0,0 +1,5 @@
+pytest
+juju
+pytest-operator==0.22.0
```

Let's pin to the most recent version to fix the issue with integration tests
```suggestion
pytest-operator==0.26.0
```

---

done